test/utils/mockAjax.js

```diff
@@ -26,6 +26,31 @@ module.exports = {
 
   },
 
+  // Find time endpoint with testmode: true
+  findTimeOnTestModeApp: function() {
+
+    var today = moment();
+    var tomorrow = moment().add(1, 'day');
+
+    jasmine.Ajax.stubRequest(
+      'https://api.timekit.io/v2/findtime'
+    ).andReturn({
+      status: 200,
+      statusText: 'HTTP/1.1 200 OK',
+      contentType: 'application/json',
+      responseHeaders: {
+        "timekit-testmode": 'true'
```

Comment:
Shouldn't this be in PascalCase so it corresponds with the API `Timekit-TestMode`?

Reply:
![screen shot 2017-09-26 at 14 34 31](https://user-images.githubusercontent.com/1449697/30860579-dfadc91e-a2c7-11e7-858e-8f134b9d00c1.png)

I get this response, so I wouldn't think so

Reply:
fixed!